build.ps1

```diff
@@ -5,21 +5,21 @@ New-Item -ItemType Directory -Force -Path ./nuget
 dotnet tool restore
```

Comment:
This is not necessary anymore - right?

Reply:
It's still required, because I didn't put the `restore-tools` target into `Program.Partial.cs`. I figured that you probably wouldn't want that same thing running five times when you run the multi-build.

This is something I'm planning to address with my ideas around improving the multi-build. That said, I could put it into `Program.Partial.cs` now, to make each sub-build self-contained, but it would come at the cost of running that same target five times during the multi-build. It's your call.

Reply:
right - makes sense!

Reply:
Signing will never be done on an individual project - so it's fine the way it is.